view/frontend/templates/js/replacejs.phtml

```diff
@@ -510,6 +530,9 @@ if (!$block->isEnabled()) return;
 
                     <?php echo $block->getjavascriptSuccess(); ?>
                     callbacks.success_url = data.success_url;
+
+                    gBoltTrack.gSuccess();
```

Comment:
try-catch so we can always call `callback`

Reply:
@daisy1754 done